### PR TITLE
Update readme to reference user shortcut

### PR DESCRIPTION
**Pull request by github-throwaway**
As requested in #120 I updated the README

### CODE REVIEW

**Comment by github-throwaway** [comment]
Any idea why the commits you already merged still show up in this PR? I just changed the readme ¯\_(ツ)_/¯

File: README.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ http://redditp.com
 * c - collapse/uncollapse controls
 * i - open image in a new tab
 * r - open comments in a new tab 
+* u - open user slideshow in new tab
```
Comment (jmdn20) [inline README.md:14]: ###

Comment (jmdn20) [inline README.md:14]: nada que previsualizar

Comment (jmdn20) [inline README.md:14]: [Uploading README.md…]()

File: README.md
Comment (jmdn20) [inline README.md:1]: Nada que previsualizar

**Review by jmdn20 — COMMENTED** [review]
Nada que previsualizar

**Pull request merged**